### PR TITLE
Fix mtoh crash and shader warnings

### PR DESCRIPTION
**Pull request by HdC-adsk**
#68 :
* Fix the crash in mtoh due to inconsistent WANT_UFE_BUILD C++ macro definition across compilation units
* Fix the warning about shader fragment connection due to name collision of shader fragments in mayaUsd and pxrUsdPreviewSurface plugin.

### CODE REVIEW

File: lib/render/vp2ShaderFragments/Float4ToFloat3.xml [hunk 1]
```diff
@@ -1,58 +1,58 @@
-<fragment uiName="float4ToFloat4" name="float4ToFloat4" type="plumbing" class="ShadeFragment" version="1.0" feature_level="0">
+<fragment uiName="Float4ToFloat3" name="Float4ToFloat3" type="plumbing" class="ShadeFragment" version="1.0" feature_level="0">
```
Comment (HdC-adsk) [inline lib/render/vp2ShaderFragments/Float4ToFloat3.xml]: git history is messed up due to file renaming. But the final xml files are correct.

**Comment by BigRoy** [comment]
I can confirm this fixes the hard maya crash of #68 and also avoids the XML warnings about the shader fragments. Nice! :)

However, I'm still seeing the following output in the Output Window:
```
Coding Error: in _Load at line 248 of D:\dev\usd\USD\pxr\base\lib\plug\plugin.cpp -- Load of 'd:/dev/usd/maya-usd-build/install/RelWithDebInfo/lib/usd/hdMaya/resources/hdMaya.dll' for 'hdMaya' failed: The specified module could not be found.


Coding Error: in _Load at line 248 of D:\dev\usd\USD\pxr\base\lib\plug\plugin.cpp -- Load of 'd:/dev/usd/maya-usd-build/install/RelWithDebInfo/lib/usd/hdMaya/resources/hdMaya.dll' for 'hdMaya' failed: The specified module could not be found.

...
```

**Comment by HdC-adsk** [comment]
Right, but that is not a regression. We need to ask Luma Pictures about that issue and see what should come next.

**Comment by HdC-adsk** [comment]
@BigRoy Please log the other issues with github. We will discuss and address them.

**Comment by suekieza** [comment]
Shared my build log in https://github.com/Autodesk/maya-usd/issues/98.
@BigRoy seems to think I should be on? What is the flag for the python build script and I will rebuild with it on, just to be sure.
Thanks

**Comment by HdC-adsk** [comment]
@suekieza I am not sure about the context, but from #98 I see there is a reply to your question.

**Pull request merged**